Produce production version for library

anbinm-msft opened the pull request:
This change provides a minified and non-minified version of the built javascript and css

msftclas commented:
Hi __@mttwc__, I'm your friendly neighborhood Microsoft Pull Request Bot (You can call me MSBOT). Thanks for your contribution!
    <span>
        In order for us to evaluate and accept your PR, we ask that you sign a contribution license agreement. It's all electronic and will take just minutes. I promise there's no faxing. https://cla.microsoft.com.
    </span>

TTYL, MSBOT;

anbinm-msft committed:
Prepare webpack config for production release

- Ignore dist folder in tsconfig
- Import style into the tsx file so webpack can generate .css .js with
  same name
- Clean the dist folder before generating prod file
- Separate dev config from prod config
- Remove commented code

anbinm-msft committed:
Add production build and bundle analyzer

- Generate .js and .min.js files
- Add support for bundle analyzer

anbinm-msft committed:
Add PostCSS loader for auto prefixing

- Remove any form of minification

anbinm-msft committed:
Simplify production configuration

- This runs webpack twice so we can produce minified and non-minified
  version easily.

anbinm-msft committed:
Create a standalone example to consume picker

Make sure we are exporting react correctly

msftclas commented:
@anbinm-msft,
Thanks for your contribution as a Microsoft full-time employee or intern. You do not need to sign a CLA.
_Thanks,
Microsoft Pull Request Bot_

anbinm-msft committed:
Merge branch 'feature/move-to-react' into move-to-react-production-build

leakymemory reviewed (approved):
Nice!
